src/ATGUI/Windows/playerlist.cpp

```diff
@@ -163,69 +164,80 @@ void PlayerList::RenderWindow()
+							Aimbot::friends.push_back(entityInformation.xuid);
+						else
+							Aimbot::friends.erase(std::find(Aimbot::friends.begin(), Aimbot::friends.end(), entityInformation.xuid));
+					}
+
+					bool shouldResolve = std::find(Resolver::Players.begin(), Resolver::Players.end(), entityInformation.xuid) != Resolver::Players.end();
+					if (ImGui::Checkbox(XORSTR("Resolver"), &shouldResolve))
+					{
+						if (shouldResolve)
+							Resolver::Players.push_back(entityInformation.xuid);
+						else
+							Resolver::Players.erase(std::find(Resolver::Players.begin(), Resolver::Players.end(), entityInformation.xuid));
+					}
 
-				if (ImGui::Button(XORSTR("Votekick")))
-				{
-					std::string cmd;
-					//													userid
-					cmd.reserve((sizeof("callvote kick ") / sizeof("")) + 3);
-					cmd.append(XORSTR("callvote kick "));
-					cmd.append(std::to_string(entityInformation.userid));
-					engine->ClientCmd_Unrestricted(cmd.c_str());
-				}
-
-				const char* clanTag = (*csPlayerResource)->GetClan(currentPlayer);
-				if (clanTag && clanTag[0] && ImGui::Button(XORSTR("Steal clan tag")))
-				{
-					Settings::ClanTagChanger::enabled = true;
-					strcpy(Settings::ClanTagChanger::value, clanTag);
-					Settings::ClanTagChanger::type = ClanTagType::STATIC;
-
-					ClanTagChanger::UpdateClanTagCallback();
-				}
-			}
-			ImGui::NextColumn();
-			{
-				if (ImGui::Button(XORSTR("Print information")))
-				{
-					cvar->ConsoleColorPrintf(ColorRGBA(255, 255, 255), XORSTR("\n=====\nPlayer informations:\n[%s] %s \nSteamID: %s\n=====\n"),(*csPlayerResource)->GetClan(currentPlayer), entityInformation.name, entityInformation.guid);
+					ImGui::NextColumn();
+					{
+						if (ImGui::Button(XORSTR("Steal name")))
+						{
+							std::string name(entityInformation.name);
+							name = Util::PadStringRight(name, name.length() + 1);
+
+							strcpy(nickname, name.c_str());
+							NameChanger::SetName(Util::PadStringRight(name, name.length() + 1));
+						}
+
+						const char* clanTag = (*csPlayerResource)->GetClan(currentPlayer);
+						if (clanTag && clanTag[0] && ImGui::Button(XORSTR("Steal clan tag")))
+						{
+							Settings::ClanTagChanger::enabled = true;
+							strcpy(Settings::ClanTagChanger::value, clanTag);
+							Settings::ClanTagChanger::type = ClanTagType::STATIC;
+
+							ClanTagChanger::UpdateClanTagCallback();
+						}
+					}
+					ImGui::NextColumn();
+					{
+						if (ImGui::Button(XORSTR("Print information")))
+							cvar->ConsoleColorPrintf(ColorRGBA(255, 255, 255), XORSTR("\n=====\nPlayer informations:\n[%s] %s \nSteamID: %s\n=====\n"),(*csPlayerResource)->GetClan(currentPlayer), entityInformation.name, entityInformation.guid);
+
+						if (localplayer && (*csPlayerResource)->GetTeam(currentPlayer) == localplayer->GetTeam())
+						{
+							if (ImGui::Button(XORSTR("Votekick")))
+							{
+								std::string cmd;
+								//													userid
+								cmd.reserve((sizeof("callvote kick ") / sizeof("")) + 3);
```

Comment:
.reserve() not needed

Reply:
I pasted this from another PR lol. Thanks!

Reply:
It's not _needed_, but it improves performance.